app/src/main/java/org/p2p/wallet/push_notifications/model/DeviceToken.kt

```diff
@@ -3,6 +3,8 @@ package org.p2p.wallet.push_notifications.model
 import com.google.gson.annotations.SerializedName
 
 data class DeviceToken(
```

Comment:
By the name it's not obvious to determine, is it Request body or it's response body, cause if it's response body i don't recommend to use default values